### PR TITLE
test: racheting coverage

### PR DESCRIPTION
**Pull request by alyssawilk**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: test/per_file_coverage.sh [hunk 1]
```diff
@@ -63,24 +62,24 @@ declare -a KNOWN_LOW_COVERAGE=(
...
 "source/extensions/transport_sockets/tls/cert_validator:95.1"
 "source/extensions/transport_sockets/tls/private_key:88.9"
 "source/extensions/wasm_runtime/wamr:0.0" # Not enabled in coverage build
 "source/extensions/wasm_runtime/wasmtime:0.0" # Not enabled in coverage build
 "source/extensions/wasm_runtime/wavm:0.0" # Not enabled in coverage build
 "source/extensions/watchdog:83.3" # Death tests within extensions
-"source/extensions/listener_managers/validation_listener_manager:69.6"
+"source/extensions/listener_managers/validation_listener_manager:70.0"
 "source/extensions/watchdog/profile_action:83.3"
-"source/server:93.3" # flaky: be careful adjusting. See https://github.com/envoyproxy/envoy/issues/15239
```
Comment (alyssawilk) [inline test/per_file_coverage.sh:76]: current coverage is "source/server:94.3 (93.3)" so I think this still leaves enough wiggle room

**Comment by alyssawilk** [comment]
Ryan I know repokitteh auto-assigns but I'll leave this as a draft until it passes and ping you when it does.

**Comment by alyssawilk** [comment]
OK, passing :-)

**Review by RyanTheOptimist — APPROVED** [review]
Awesome!

**Pull request merged**